xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-ui/src/main/resources/XWiki/QuickSearchUIX.xml

```diff
@@ -118,19 +118,32 @@
     </property>
     <property>
       <code>require(['jquery', 'bootstrap'], function($) {
+
+  // Returns true if the window inner width is lower than 768.
+  function isSmallScreen() {
+    var windowWidth = window.innerWidth;
+    return windowWidth &lt; 768;
```

Comment:
768 could have also be a parameter, since @screen-sm-min is configurable, and since it's used in at least 2 places.

Reply:
Thanks for the review, @evalica. Do you mean that we could use @screen-sm-min directly from JavaScript or that we should introduce a variable in the JavaScript itself that we would initialize to 768, or something else?

Reply:
> Do you mean that we could use @screen-sm-min directly from JavaScript or that we should introduce a variable in the JavaScript itself that we would initialize to 768, or something else?

I guess that if it's configurable then you have a way to retrieve it with a velocity call, so you could put that value in the html as a data-attribute and reuse it in your javascript probably.

Reply:
Thank you Simon. As far as I can see, the screen dimension breakpoints are configurable indeed as variable declarations in the FlamingThemes directly. I could not find a way to retrieve their value from Velocity though. Hence I tried another approach in the commits below: add a CSS class on the parent `ul` element of the search input reflecting the search input state (open/close) and control the with of the input and the display of its adjacent buttons entirely in CSS using this new class. Also updated `suggestSearch.js` and `suggest.css`. The latter has no access to the screen dimension variables so I used a hardcoded width in that one, which is the case already for other CSS files such as `columns.css`, for which we could switch to LESS files I guess.